src/editor/_main.cpp

```diff
@@ -68,7 +68,9 @@ int
 main(int argc, char *argv[])
 {
     Tilemap tilemap;
+    Tileset tileset;
     LoadFileResult load_result = Tilemap_load_from_file("tilemap.json", tilemap);
```

Comment:
change the name of this `load_result` to `load_tilemap_result` to match the newly added load result for tileset.